swarmer/src/main/kotlin/com/gojuno/swarmer/Emulators.kt

```diff
@@ -299,8 +299,12 @@ private fun waitForEmulatorToFinishBoot(targetEmulator: Emulator): Observable<Em
 
 private fun Long.nanosAsSeconds(): Float = NANOSECONDS.toMillis(this) / 1000f
 
-private fun outputFileForEmulator(args: Commands.Start) = File("${args.emulatorName}.output")
-        .apply { deleteOnExit() }
+private fun outputFileForEmulator(args: Commands.Start) =
+        File(
+                args.redirectOutputTo ?: "",
+                "${args.emulatorName}.output"
+        )
+                .apply { if (!args.keepOutputOnExit) deleteOnExit() }
```

Comment:
Formatting seems wrong.

Reply:
Hmm, you mean the indent?

Reply:
```kotlin
private fun outputFileForEmulator(args: Commands.Start) = File(args.redirectOutputTo ?: "", "${args.emulatorName}.output").apply {
    if (!args.keepOutputOnExit) deleteOnExit()
}
```
or
```kotlin
private fun outputFileForEmulator(args: Commands.Start) =
    File(args.redirectOutputTo ?: "", "${args.emulatorName}.output").apply {
        if (!args.keepOutputOnExit) deleteOnExit()
    }
```
Not sure what auto-formatter will do.

Reply:
Ok with 2 one

Reply:
Another one:
```kotlin
private fun outputFileForEmulator(args: Commands.Start) =
        File(
                args.redirectOutputTo ?: "",
                "${args.emulatorName}.output"
        ).apply {
            if (!args.keepOutputOnExit) deleteOnExit()
        }
```